README.md

```diff
@@ -93,6 +93,35 @@ SELECT COUNT(jid) FROM sidekiq_jobs WHERE event = 'error' AND time > now() - 1d
 
 Et cetera.
 
+### Stats and Queues metrics
+
+To collect metrics for task stats and queues, you need to run the script periodically. For example you can use gem `clockwork`.
+
+For stats metrics:
+```ruby
+require "sidekiq/metrics/stats"
+
+Sidekiq::Metrics::Stats.new(
+            influxdb_client: InfluxDB::Client.new(options), # REQUIRED
+            series_name: 'sidekiq_stats',                   # optional, default shown
+            retention_policy: nil,                          # optional, default nil
+            tags: {},                                       # optional, default {}
+          ).call
+```
+
+For queues metrics:
+```ruby
+require "sidekiq/metrics/queues"
+
+Sidekiq::Metrics::Queues.new(
+            influxdb_client: InfluxDB::Client.new(options), # REQUIRED
+            series_name: 'sidekiq_stats',                   # optional, default shown
+            retention_policy: nil,                          # optional, default nil
+            tags: {},                                       # optional, default {}
+          ).call
+```
+
+
```

Comment:
Please add a description of the series that the user will have in their database. Which tags and values will be added.